### PR TITLE
UPDATE: InputStepper accessibility

### PR DESCRIPTION
**Pull request by filipdanisko**
- Updating focus on a buttons to be visible
- Moving to prevent default in the key handler to an inside of a condition to prevent focus trap.<br/><br/><br/><url>LiveURL: https://orbit-components-update-inputStepper.surge.sh</url>

### CODE REVIEW

File: src/InputStepper/index.js [hunk 1]
```diff
@@ -118,24 +151,28 @@ class InputStepper extends React.Component<Props & ForwardedRef, State> {
...
           prefix={
-            <ButtonLink
-              disabled={disabled || value <= +minValue}
-              iconLeft={<MinusCircle color="secondary" />}
-              size={size}
-              onClick={this.decrementCounter}
-              transparent
-              component={PrefixSuffix}
-            />
+            <StyledButtonWrapper role="button" tabIndex="0" onKeyDown={this.handleKeyDecrement}>
```
Comment (vepor) [inline src/InputStepper/index.js]: Shouldn't be this handled by `ButtonLink` itself? I mean all properties (can be added, make sense for me) and also the focus/active state.

Comment (filipdanisko) [reply src/InputStepper/index.js]: There are no types for that, so this is a workaround. Should I add the types?

File: src/InputStepper/index.js [hunk 2]
```diff
@@ -78,6 +99,18 @@ class InputStepper extends React.Component<Props & ForwardedRef, State> {
     }
   };
 
+  handleKeyDecrement = (ev: SyntheticKeyboardEvent<HTMLInputElement>) => {
+    if (ev.keyCode === 32 || ev.keyCode === 13) {
```
Comment (vepor) [inline src/InputStepper/index.js]: Please add the key codes into `keyMaps.js` 🙏

File: src/InputStepper/index.js (hunk 1 above)
Comment (vepor) [reply src/InputStepper/index.js]: Well, we can use your solution for now. But it make sense to have these props available in Button/ButtonLink component as other devs can use them inside their projects.

Comment (filipdanisko) [reply src/InputStepper/index.js]: @vepor I agree but I would rather do that in a separate PR.

**Pull request merged**